Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -14,16 +14,128 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
         }
 
-        public static int FindDigit(string equation)
-        {
+        public static int FindDigit(string s)
+        {
             // Add your code here.
-            throw new NotImplementedException();
+            
+            int r = -1;
+            int a=0, b=0, c=0;
+            String A="", B="", C="";
+            int strt = s.IndexOf("*");
+            int lst = s.IndexOf("=");
+            
+            //Distributing the no. in A B C
+            for(int i=s.Length-1; i>=0; i--){
+                if(s[i].Equals('=')){
+                    C = s.Substring(i+1);

```

Comment:
can you do it without for loop?
you already have `strt` and `lst`

Reply:
removed for loop